### DISCUSSION TITLE
Welcome to PHPAuth Discussions!

### DISCUSSION THREAD

**Discussion by Conver** [PHPAuth/PHPAuth#503] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KarelWintersky** [comment]
Creating new code, fixing legacy bugs :)

**Comment by louis123562** [comment]
Hey,

my name is Louis and i am a developer (mainly web, cloud and app).

I use phpAuth in two big projects with more than 2.000 users so far, which all authenticate with it.

The greatest thing about phpAuth is, that it is very lightweight (only two source files, yay) like FPDF.

The current version is good for basic authentication, but we should go forward and add extensions for authentication via OAuth like from Google, Micorosoft, Apple and so on, so that phpAuth can take the part of session handling as it also does so far.

In V2 it is also possible to use a JWT instead of a cookie, which is also a greate feature. But the implementation sticks - for my opinion - to deep in the source code instead of using it like a plugin or configuration, as many already mentioned.

I think we all need more time in our life, im busy as f as well at the moment, not only because of covid, but there is much to do.

Keep healthy and in the near future we will push phpAuth much more!
